### PR TITLE
Updated documentation for Vuetify element: panels

### PR DESCRIPTION
**Pull request by lyalchenko1995**

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -878,33 +861,112 @@ It is **necessary** to specify **the root** of an element.
 
 ExpansionPanels is a list of ExpansionPanel.
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:862]: Maybe to put this phrase at the end of description?:)

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -878,33 +861,112 @@ It is **necessary** to specify **the root** of an element.
...
 **content()** | Expands if panel is closed. Returns content element by content locator from JDIExpansionPanels annotation| UIElement
 **expand()** | Expands panel | void
 **close()** | Closes panel | void
 **isExpanded()** | Returns true if panel is expanded | boolean
 **isClosed()** | Returns true if panel is closed | boolean
+**contentColor()** | Returns panel header font color | String
+**contentBackgroundColor()** | Returns panel content background color | String
+**hasIconDisableRotate()** | Check that panel header has icon disable rotate | boolean
+
+In addition, ExpansionPanel implements HasColor.
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:968]: here we also put assert methods ExpansionPanelAssert and ExpansionPanelsAssert

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md [hunk 3]
```diff
@@ -878,33 +861,132 @@ It is **necessary** to specify **the root** of an element.
...
-and [JDI vuetify page tests for expansion panels](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/ExpansionPanelsTest.java).
+| Assert method | Description |
+| :--- | :--- |
+**expanded()** | Assert that panel is expanded
+**closed()** | Assert that panel is closed
+**contentColor(String color)** | Assert that panel has content font color '{0}'
+**contentBackgroundColor(String color)** | Assert that panel has content background color '{0}'
+**iconDisableRotate()** | Assert that panel has icon disable rotate
+**notIconDisableRotate()** | Assert that panel icon rotation is not disabled
+
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:988]: panelS asserts also should be here :)

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 3 above)
Comment (lyalchenko1995) [reply source/includes/documentation/_vue_js_elements.md:988]: They are above

**Review by MayaElf — APPROVED** [review]

**Review by vklonin — APPROVED** [review]

**Pull request merged**